src/tsolvers/lasolver/LASolver.cc

```diff
@@ -807,6 +807,7 @@ void LASolver::printStatistics(std::ostream & out) {
     laSolverStats.printStatistics(out);
 }
 
+//? TK: I do not understand this, is it related directly to inter. or to resolution proofs?
```

Comment:
Not directly connected to resolution proofs.
This condition should stay as is, I believe.